cypress/integration/lpa/donors/donor.spec.js

```diff
@@ -0,0 +1,52 @@
+describe("Create Donor", { tags: ["@lpa", "@smoke-journey"] }, () => {
+  beforeEach(() => {
+    cy.loginAs("Case Manager");
+    cy.visit('/lpa/home');
+  });
+
+  it("should create a new donor", function () {
+    cy.intercept({ method: 'GET', url: '/api/v1/assignees/*/tasks*' }).as('tasksRequest');
+    cy.wait('@tasksRequest');
+
+    cy.get("uib-tab-heading[id=Timeline]").contains("Timeline").click();
+    cy.contains("Create Donor").click();
+
+    cy.get(".action-widget-content").within(() => {
+      cy.get("#firstname0").type("Spongebob");
+      cy.get("#surname0").type("Squarepants");
+      cy.contains("Save and Exit").click();
+    });
+
+    cy.contains(/Person (\d+(-|)){3} was created/).click();
+    cy.get(".timeline-event").last().should("contain", "Person (Create / Edit)");
```

Comment:
it would probably be a good idea to have `cy.get('.timeline .timeline-event', { timeout: 10000 })` before this too